ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -429,11 +431,15 @@ objects:
         fi
         if [[ "${CLUSTER_TYPE}" = "aws" ]]; then
             master_type=null
+            worker_type=m5.4xlarge
             if [[ "${CLUSTER_VARIANT}" =~ "xlarge" ]]; then
               master_type=m5.8xlarge
             elif [[ "${CLUSTER_VARIANT}" =~ "large" ]]; then
               master_type=m5.4xlarge
             fi
+            if [[ "${CLUSTER_VARIANT_WORKER}" =~ "xlarge" ]]; then
+              worker_type=m5.8xlarge
+            fi
```

Comment:
still no :(

```
elif $cluster_variant =~ "xlarge_worker"; then
  blah
fi
```
is what I had in mind, although you'll also have to fix the above checks because they look for any string containing "xlarge" which means if you set the varant to "xlarge_worker" they will also set the master to xlarge.  so we need something more explicit.

Reply:
@bparees does this check work better? then it doesn't affect the previous logic and still allows setting worker type?

Reply:
it's better, but when you set "xlarge_worker" the "`=~ "xlarge` logic above will stil be triggered, so you're also going to get xlarge masters even though you don't need them.  so that's not ideal.  We need to find someone who knows more about how these values are set (I think cluster bot uses them) to see if we can tighten the `=~ "xlarge"` logic above so that we can distinguish between "xlarge"(implies masters) and "xlarge_worker"

Reply:
if you're really blocked right now, let's just for the template and hardcode what you need, update your job to use the new template, and we'll fix it later.  i don't want you to miss feature cut due to futzing w/ this.

Reply:
@bparees new change should be best of both worlds